src/rapidata/rapidata_client/order/dataset/_rapidata_dataset.py

```diff
@@ -28,81 +25,36 @@ class RapidataDataset:
     def __init__(self, dataset_id: str, openapi_service: OpenAPIService):
         self.id = dataset_id
         self.openapi_service = openapi_service
-        self.local_file_service = LocalFileService()
+        self.datapoint_uploader = DatapointUploader(openapi_service)
 
     def add_datapoints(
```

Comment:
this method seems obsolete, I guess it can be merged with the _process_uploads_in_chunks method as it's just a proxy now